framework/include/restart/DataIO.h

```diff
@@ -374,7 +374,7 @@ dataLoad(std::istream & stream, std::list<T> & l, void * context)
   {
     T data;
     loadHelper(stream, data, context);
-    l.push_back(data);
+    l.push_back(std::move(data));
```

Comment:
I need to go through all the DataIO methods and mark many of them with `std::move`...